src/parachain/escrow.ts

```diff
@@ -287,16 +292,21 @@ export class DefaultEscrowAPI implements EscrowAPI {
         };
     }
 
-    async getEscrowStake(accountId: AccountId): Promise<Big> {
+    async getEscrowStakeRaw(accountId: AccountId): Promise<Big> {
         const rawStake = await this.api.query.escrowRewards.stake(accountId);
         return decodeFixedPointType(rawStake);
     }
 
-    async getEscrowTotalStake(): Promise<Big> {
+    async getEscrowTotalStakeRaw(): Promise<Big> {
         const rawTotalStake = await this.api.query.escrowRewards.totalStake();
         return decodeFixedPointType(rawTotalStake);
     }
 
+    async getEscrowTotalStake(): Promise<MonetaryAmount<Currency<GovernanceUnit>, GovernanceUnit>>  {
+        const rawTotalStake = await this.getEscrowTotalStakeRaw();
+        return newMonetaryAmount(rawTotalStake, this.governanceCurrency as Currency<GovernanceUnit>);
```

Comment:
The total stake in the reward pool are not KINT tokens. The stake in the reward pool are vKINT, i.e., the staked KINT tokens multiplied by the staking duration. To get the total staked KINT, you need to use:
`escrow.locked.entries()` and sum them up.